torch/testing/_internal/logging_tensor.py

```diff
@@ -22,6 +22,13 @@ def no_dispatch() -> Iterator[None]:
 # 3. Enter dispatcher, wind your way through Autograd
 # 4. Hit Python dispatch key, call __torch_dispatch__
 
+# This Tensor can work with autograd in two ways:
```

Comment:
Update this to say that allowing both is for testing only. And you should not mix the two in a single test or you're going to get very weird behavior!

Reply:
Question: are you actually planning on using LoggingTensor in both ways for testing?

Reply:
Yes. The current test do use the logging Tensor having gradients.
And the new ones added above in this stack check for the wrapped Tensor requirering gradients